params/config.go

```diff
@@ -531,6 +531,14 @@ type ScrollConfig struct {
 
 	// L1 config
 	L1Config *L1Config `json:"l1Config,omitempty"`
+
+	// System contract config
+	SystemTxConfig *SystemTxConfig `json:"systemTxConfig,omitempty"`
+}
+
+type SystemTxConfig struct {
+	Signers   []common.Address `json:"signers"`
```

Comment:
Is this list defined at the beginning and not updated anymore?

If it's updated, then it's better to store in in state (in a system contract) so that updates are part of the replicated state machine.

Also, if a follower node changes this config, that would result in a fork at some unexpected point in the future.

Reply:
Yeah, we need to exercise some care here -- probably storing in state (system contract) is best.

Reply:
best solution: use a static list based on height/fork; will do in follow up pr.